python/ray/tune/ray_trial_executor.py

```diff
@@ -91,10 +146,32 @@ def noop_logger_creator(config, logdir):
     return NoopLogger(config, logdir)
 
 
-# The class responsible for communicating back to TrialRunner.
-class FutureResult:
-    def __init__(self, type, trial=None, result=None):
-        self.type = type
+class ExecutorEventType(Enum):
+    """The executor event type.
+
+    Some of the events are internal events to executor while others
+    are handled by runner."""
+
+    NO_RUNNING_TRIAL_TIMEOUT = 1
```

Comment:
we can use `enum.auto()` here, but this is fine too

Reply:
noted. A good trick to know :) Thanks!